scripts/webpack/prod.client.config.js

```diff
@@ -121,7 +121,10 @@ module.exports = ({isDeploy, isStats}) => ({
       __APP_VERSION__: JSON.stringify(process.env.npm_package_version),
       'process.env.DEBUG': false,
       'process.env.NODE_ENV': JSON.stringify('production'),
-      'process.env.PROTOO_LISTEN_PORT': JSON.stringify((process.env.PROTOO_LISTEN_PORT || 4444) - 1)
+      'process.env.PROTOO_LISTEN_PORT': JSON.stringify((process.env.PROTOO_LISTEN_PORT || 4444) - 1),
+      'process.env.AUTH_INTERNAL_ENABLED': process.env.AUTH_INTERNAL_DISABLED !== 'true',
```

Comment:
-1 we want to put these in https://github.com/ParabolInc/parabol/blob/eb295331feba7e2534c48012d252c2a52b802905/packages/server/createSSR.ts for production.

the reason why is we ultimately want to move to a build-once-run-everywhere infrastructure.
If we left them here, we'd have to build once per environment.
if we put them in createSSR, we could use the same build for privately managed instances, the cloud offering, etc. Plus, if we ever wanted to e.g. disable google auth, we could do that by redeploying the app vs. rebuilding it 🎉